storage/client.go

```diff
@@ -48,6 +49,10 @@ const (
 	userAgentHeader = "User-Agent"
 )
 
+var (
+	validStorageAccount = regexp.MustCompile("^[0-9a-z]{3,24}$")
```

Comment:
Upon further consideration, it get's in the way too much of just calling a function. I'll type up another way to do this.